requirements-dev.txt

```diff
@@ -8,9 +8,9 @@ craft-application==3.2.0
 craft-archives==1.2.0
 craft-cli==2.6.0
 craft-grammar==1.2.0
-craft-parts==1.33.0
+craft-parts==1.34.0
 craft-platforms==0.1.1
-craft-providers==1.24.1
+craft-providers==1.25.0
```

Comment:
Be prepared for reports about `multipass launch snapcraft:core24` requiring multipass 1.15 (on the edge channel)